avocado_cloud/app/nutanix/nutanix.py

```diff
@@ -191,7 +192,82 @@ def create_vm(self, ssh_pubkey=None):
         }
 
         return self.make_request(endpoint, 'post', data=data)
-
+       
+    def create_vm_ISO_kickstart(self):
+        logging.debug("Create VM by ISO kickstart")
+        endpoint = urljoin(self.base_url, "vms")
+	# Attach image.
+        images = self.list_images()
+        vmdisk_uuid = []
+        for image in images['entities']:
+            if self.image_name_iso == image['name']:
+                vmdisk_uuid.append(image['vm_disk_id'])
+            if self.image_name_kickstart_iso == image['name']:
+                vmdisk_uuid.append(image['vm_disk_id'])
+        logging.debug("vmdisk_uuid is "+str(vmdisk_uuid))
+        if vmdisk_uuid == "":
```

Comment:
Need to check both installation iso and kickstart iso exist

Reply:
fixed